src/screens/SendBitcoin/SendBitcoinConfirm.js

```diff
@@ -99,22 +118,28 @@ class SendBitcoinConfirm extends React.Component<Props, State> {
         >
           <LabeledRow>
             <Label>Amount</Label>
-            <Value>{amount} BTC</Value>
+            <Value>{amount} BTC ({formattedAmountInFiat})</Value>
```

Comment:
amount sometimes gets rendered with decimals for input 0.003 i get 
<img width="321" alt="Screen Shot 2019-12-20 at 16 00 46" src="https://user-images.githubusercontent.com/37015091/71263446-2ee82180-2342-11ea-8c0d-6c248d152add.png">